package.json

```diff
@@ -9,6 +9,7 @@
     "backstop:test:ci": "backstop test --config=backstop.config.js",
     "backstop:approve": "backstop approve --config=backstop.config.js",
     "build": "npm-run-all -l clean build:*",
+    "build-cms": "node ./build-cms.js -w",
```

Comment:
I think this is a bit confusing because we have a `build:cms` script too, and all the `build:*` commands are for building the site, not running in dev mode. How about you change this to `cms` and then change `cms` to `proxy`?

Reply:
Yep, that makes sense; thanks for the suggestion.